includes/class-edd-download.php

```diff
@@ -666,19 +666,13 @@ public function get_sales() {
 	 */
 	public function increase_sales( $quantity = 1 ) {
 
-		$quantity    = absint( $quantity );
-		$total_sales = $this->get_sales() + $quantity;
+		_edd_deprecated_function( __METHOD__, '3.0', 'EDD_Download::recalculate_net_sales_earnings()' );
+		edd_recalculate_download_sales_earnings( $this->ID );
 
-		if ( $this->update_meta( '_edd_download_sales', $total_sales ) ) {
+		$this->get_sales();
```

Comment:
I've updated these to all be marked as deprecated, and run the recalculation functions as needed. The action hooks are still available.